blocks/variables_dynamic.js

```diff
@@ -45,48 +45,48 @@ Blockly.Constants.VariablesDynamic.HUE = 310;
 Blockly.defineBlocksWithJsonArray([ // BEGIN JSON EXTRACT
   // Block for variable getter.
   {
-    "type": "variables_get_dynamic",
-    "message0": "%1",
-    "args0": [{
-      "type": "field_variable",
-      "name": "VAR",
-      "variable": "%{BKY_VARIABLES_DEFAULT_NAME}"
+    'type': 'variables_get_dynamic',
```

Comment:
Ack, please undo your find/replace all. The JSON snippets must use double quotes. They are an exception to the standard JS style which uses single quotes.